civic-tech-club/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Use the official Node.js image as the base image
-FROM node:18-alpine
+FROM node:22-alpine
```

Comment:
LOVE it! for reference, node 22 is not the current LTS version, but that's fine. This will prep us to opt in once 22 becomes LTS.